koku/masu/processor/ocp/ocp_report_db_cleaner.py

```diff
@@ -135,7 +129,7 @@ def purge_expired_report_data_by_date(self, expired_date, simulate=False):
                 LOG.info(log_json(msg="deleted table partitions", count=del_count, schema=self._schema))
 
                 # Remove all data related to the report period
-                del_count, _ = OCPUsageReportPeriod.objects.filter(id__in=all_report_periods).delete()
+                del_count = cascade_delete(all_usage_periods.query.model, all_usage_periods)
```

Comment:
the `cascade_delete` function has no return, i think `del_count` here will be `None` and logging it maybe misleading.

Reply:
removed! Good catch.